picard/ui/options/dialog.py

```diff
@@ -170,6 +170,12 @@ def _restore_default(self, page):
             if option.section == 'setting':
                 old_options[option.name] = config.setting[option.name]
                 config.setting[option.name] = option.default
+
+        # Plugin manager has to be updated
```

Comment:
we can have a load_default method method implemented. I guess, I'll update the implementation and add a load_defaults method.